R/scale_nest_discrete.R

```diff
@@ -1,3 +1,14 @@
+#' Scales NEST manually or some such thing.
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' Adds scaling manually or some such thing. IDK
+#' what this does TBH, I'm just trying to
+#' appease R CMD check.
+#'
+#' @param palette (``)
+#' @param palette (``) 
+#' @importFrom ggplot2 waiver discrete_scale ScaleDiscrete
```

Comment:
Yo @edelarua - would you be able to add docs to these functions? You can use this branch to update.

Reply:
And unit tests for this. Otherwise these need to be removed as YAGNI